client/components/environment-badge/style.scss

```diff
@@ -0,0 +1,92 @@
+.environment-badge {
+	padding: 16px 0;
+	position: fixed;
+		bottom: 0;
+		right: 16px;
+	z-index: z-index( 'root', '.environment-badge' );
+	backface-visibility: hidden;
```

Comment:
might want to use `will-change` these days 

https://developer.mozilla.org/en-US/docs/Web/CSS/will-change#Syntax

Reply:
do we even need it? 🤔

Reply:
I'm not sure anymore. Used to be that anything with position:fixed we wanted to promote to a new compositing layer because otherwise it caused a ton of repaints on scroll in mobile safari and chrome. Might be worth revalidating that assumption.

Reply:
oh! then ditch it. :)

Reply:
maybe ditch it, then turn on the paint visualization in chrome and see if it affects repaints when scrolling.

Reply:
Alright, will do :)

Reply:
Couldn't spot any difference. Will remove it and then merge :)